feature-libs/user/profile/components/update-email/my-account-v2-email.component.html

```diff
@@ -0,0 +1,128 @@
+<cx-spinner
+  class="overlay"
+  *ngIf="(isUpdating$ | async) || !(user$ | async); $"
+>
+</cx-spinner>
+<div>
+  <div class="">
+    <div class="row">
+      <div class="col-sm-10">
```

Comment:
[MEDIUM]
New Spartacus features, should not use Bootstrap, but custom styles based on flexbox.
cc @developpeurweb

Note: This comment applies also to other HTML files in this PR.
Note: it's a continuation of another [comment](https://github.com/SAP/spartacus/pull/18348#discussion_r1473311971), which was resolved due to the other file being removed. However we still have other HTML files in this PR which use Bootstrap styles.

Reply:
Fixed. Please check.

BTW: where is the guideline for it.